### ISSUE TITLE
Presentation peer review

### ISSUE THREAD

**Issue by mfarmani95** [INFO523-S24/project-final-KGCompetitors#7]
The following is the peer review of the Presentation proposal by Analytical Avengers. The team members that participated in this review are 

- Sunday Usman - @sundayusman
- Tanya Evita George - @tanyageorge
- Mellika Akbarsharifi - @mellica97
- Divya Liladhar Dhole - @Divyadhole
- Mohammad Ali Farmani - @mfarmani95
- H M Abdul Fattah - @hmfattah
- Gabriel Gedaliah Geffen - @gabegef

- Content: Is the research question well designed and is the data being used relevant to the research question?
They stated that they would evaluate model performance, but they used only one metric. It would be good to see the performance of the model using different metrics, including the time it takes for the model to make predictions. They have provided an extensive description of the dataset, including how PII is labeled and how tokenization is handled, which is vital for anyone looking to understand or reproduce your work.

They said they would develop the model, but they did fine-tuning, which is hard to locate in their code files.

- Content: Did the team use appropriate visualizations and did they interpret them accurately?

They interpreted them accurately, but there was more opportunity to create additional plots to capture more aspects of the problem.
The use of visualizations (bar graphs) and detailed metrics (F5-scores) to compare model performance helps in understanding the efficacy of different models in a quantifiable manner.

- Creativity and Critical Thought: Is the project carefully thought out? Are the limitations carefully considered? Does it appear that time and effort went into the planning and implementation of the project?

It seems they did a report on the performance of some pre-trained models but failed to appropriately explain the methodology instead of simply reporting it

- Slides: Are the slides well organized, readable, not full of text, featuring figures with legible labels, legends, etc.?
The slides are good but lack visualizations.

- Professionalism: How well did the team present? Does the presentation appear to be well practiced? Are they reading off of a script? Did everyone get a chance to say something meaningful about the project?
They could have spoken loudly, but overall they did well.

**Comment by remisublette** [comment]
Thanks for your feedback!
